beacon-chain/p2p/gossip_scoring_params.go

```diff
@@ -0,0 +1,175 @@
+package p2p
+
+import (
+	"math"
+	"strings"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+const beaconBlockWeight = 0.8
+const aggregateWeight = 0.5
+const attestationTotalWeight = 1
+const decayToZero = 0.01
```

Comment:
Can we, please, have this combined into a single `const(...)` and, more importantly, add some comments for each value?